### PR TITLE
Student cloning instructions

### PR DESCRIPTION
**Pull request by KatherineCox**
Similar to #169, here's an attempt at student instructions for cloning.

This lets you configure the Workspace name in the parent document, and then inserts the name in the student instructions.  If this seems overly complicated, we can drop it.  Partly I just wanted to see if this would even work, and, hey, it does! :)

### CODE REVIEW

**Review by ehumph — APPROVED** [review]
My only suggestion is to make the slides/graphics showing the workspaces a bit bigger (if that's possible). Otherwise it looks good!

**Comment by avahoffman** [comment]
LOVE THIS!!!! 

My only concern is that the screenshots will always refer to the SARS workspace, even if the instructor specifies some other workspace. Could that potentially be confusing? Maybe some text like "in the example below we are finding the 'SARS Genome' Workspace" ... something like that?

**Comment by KatherineCox** [comment]
image sizing - done!  thanks @avahoffman  for saving me a Google search :)

I've added :
> (The images below show the SARS-CoV-2-Genome Workspace as an example, but you should look for the Workspace  **`r workspace_name`**.)

to try to address potential confusion.  I agree showing the SARS workspace is non-ideal, wish I could magically update the images too.

Perhaps at some point it would be good to redo the screenshots with a workspace named something like "Example_Student_Workspace", to make it more obvious that it's just an example.  Then we could also put something in the Dashboard for that workspace saying "Hey...did you really mean to clone me?".

(I actually considered doing this just now, but then hesitated because we'd need to make it public so that it shows up under the PUBLIC tab, so it probably needs a bit more thought/effort beyond just giving it a name)

**Comment by avahoffman** [comment]
Great! I have a few more comments.

I made a PR here: #174 
- First thing is the ability to add an "example" div box. I think it helps (me at least) visualize where the borrowed text starts and stops
- Third chunk is missing the module settings
- If you define `AnVIL_module_settings` in one chunk, they persist to the next chunk used. This could be a problem if someone sets variables on chunk 1 but doesn't want them on chunk 2. Added a comment to make that clear.

File: config_automation.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ url-checker: yes
 # Render preview of content with changes (Rmd's and md's are checked)
 render-preview: yes
 # Spell check Rmds and quizzes
-spell-check: yes
+spell-check: no
```
Comment (avahoffman) [inline config_automation.yml:10]:
```suggestion
spell-check: yes
```

**Comment by KatherineCox** [comment]
@avahoffman I was able to set `AnVIL_module_settings` to NULL at the end of the child using `<<-`.  There's a demo of it working in #175, but the PR was getting messy with the adjustments I had to make to get it to render a preview, so I went ahead and made the changes in this branch.  Look okay?

**Comment by avahoffman** [comment]
@KatherineCox sounds great! Let's remove these instructions, then: 
https://github.com/jhudsl/AnVIL_Template/blob/e64e48f579e9ffab75f0c43df13dba4b26735af4/08-student_modules.Rmd#L39

**Comment by KatherineCox** [comment]
> @KatherineCox sounds great! Let's remove these instructions, then:
> 
> https://github.com/jhudsl/AnVIL_Template/blob/e64e48f579e9ffab75f0c43df13dba4b26735af4/08-student_modules.Rmd#L39

Oh, good catch, thanks!  Done.

File: assets/AnVIL_style/style.css [hunk 2]
```diff
@@ -303,6 +303,19 @@ div.reflection{
   background-color: #e8ebee;
   background-image: url("../assets/box_images/thinking_face.png");
 }
+div.example{
```
Comment (KatherineCox) [inline assets/AnVIL_style/style.css:306]: @avahoffman can we delete div.example? or did you want it for something?

Comment (avahoffman) [reply assets/AnVIL_style/style.css:306]: yes let's delete it!

**Review by avahoffman — APPROVED** [review]

**Pull request merged**